packages/react-link/src/components/Link/Link.test.tsx

```diff
@@ -26,24 +27,24 @@ describe('Link', () => {
   });
 
   it('renders as a button if no href is provided', () => {
-    wrapper = mount(<Link>This is a link</Link>);
-    const button = wrapper.find('button');
-    const anchor = wrapper.find('a');
+    const component = renderer.create(<Link>This is a link</Link>);
```

Comment:
This is why deduplicating dev deps is scary...*something* that updated (I don't know what) changed the snapshot output to include `[Function anonymous]` instead of `[Function]`. Based on search results about related problems from a couple years ago, I was concerned that snapshots with generated function names might change between Node versions, so I switched these tests to use `react-test-renderer` which seems unaffected.

Reply:
nice one, hopefully we will replace enzyme with react-testing-library for more robust tests and as you mentioned test-renderer for corner cases like this one. thx!

Reply:
> nice one, hopefully we will replace enzyme with react-testing-library.
> 
> * with that said why not just use directly react-testing-lib to do the testing the "recommended way" (this is a converged component right ?)

Most of the converged components are still using react-test-renderer for snapshots, so that was easier for a quick fix.

> sidenote: just realised when I saw that you bumped babel versions as well - there might have been some changes how anonymous functions are being transpiled - but I'd have to check deeper into enzyme.

The babel version bump was mainly to allow as many things as possible (especially general compilation and storybook) to use the same version. It might have been possible to stay on 7.12, but since some of the deps had already implicitly upgraded to 7.14 per the lock file it would have been a huge pain to get them all back down to 7.12 by hand.

I did look at some of the enzyme deps (and their deps) to see if I could find anything obvious that might have caused a change to function printing in snapshots, but couldn't find anything immediately obvious and decided it probably wasn't worth the time to keep digging. My guess was that it was a difference just in printing not transpiling (and specific to the way enzyme renders stuff) since based on [this](https://jestjs.io/blog/2016/10/03/jest-16#snapshot-updates) it seems like function names are supposed to be omitted, but I could be wrong.